Kudu.Console/Program.cs

```diff
@@ -283,7 +284,8 @@ private static IEnvironment GetEnvironment(string siteRoot, string requestId)
                 repositoryPath,
                 requestId,
                 Path.Combine(AppContext.BaseDirectory, "KuduConsole", "kudu.dll"),
-                null);
+                null,
+                new FileSystemPathProvider(new NullMeshPersistentFileSystem()));
```

Comment:
Does consumption functions support Local Git Deploys, this is called as a Git POST-Receive hook to stream the build output logs to the git client. In that case would the Oryx Build expected this path to be present ? (I'm not sure if ArtifactsPath could be present on `/kudu-mnt`)

Reply:
Local Git deploy is not supported in consumption. ArtifactsPath will never be under /kudu-mnt and I think Oryx doesn't access Artifacts folder directly. Oryx produces build output under /site/wwwroot and later as a post build step we use that build output to build a squashfs file under Artifacts folder and upload to storage. So nothing should change for Oryx. ( @Hazhzeng Please verify my comment :) )

Reply:
Yes, Bala's right, we don't support Git deploy in Linux Consumption. For oryx build, it will build the package from /tmp/zipdeploy/extracted, and generated the built content into **/site/wwwroot**. After that, the SetupLinuxConsumptionFunctionAppDeployment step will zip update all content from **/site/wwwroot** into **/site/deployment** and upload the artifact to customer's storage account.

Build pipeline:
/tmp/zipdeploy/extracted => /site/wwwroot/ => /site/deployment => customer storage

```
Oryx Version      : 0.2.20200114.13, Commit: 204922f30f8e8d41f5241b8c218425ef89106d1d, ReleaseTagName: 20200114.13
Build Operation ID: |ETigkkgjbb4=.48f45d84_
Repository Commit : 09032b09e9a144f78a9cc005f97de358


Source directory     : /tmp/zipdeploy/extracted
Destination directory: /home/site/wwwroot
```